arp.py

```diff
@@ -0,0 +1,74 @@
+from ethernet import ethernet_frame
+import struct
+from binascii import unhexlify, hexlify
+
+FORMAT = ">HHbbH6s4s6s4s"
+FRAME_SIZE = struct.calcsize(FORMAT)
+HARDWARE_TYPE = 1
+PROTOCOL_TYPE = 0x800
+HARDWARE_SIZE = 6
+PROTOCOL_SIZE = 4
+REQUEST = 1
+REPLY = 2
+BROADCAST_BYTES = b'\xff\xff\xff\xff\xff\xff'
+BROADCAST_STRING = "ff:ff:ff:ff:ff:ff"
+ARP_ETHER_TYPE = 0x806
+
+
+class ARP_Frame:
```

Comment:
Classes use PascalCase convention, and variables use snake_case convention
```suggestion
class ARPFrame:
```